content/en/_index.html

```diff
@@ -29,9 +29,9 @@ <h3 class="section-head">
       <span class="border-bottom pb-2">What is Kubeflow?</span>
     </h3>
     <p class="mx-auto col-11 col-xl-7 px-0">
-      Kubeflow makes artificial intelligence and machine learning simple, portable, and scalable.
+      Kubeflow is the foundation of tools for AI platforms on Kubernetes. Kubeflow is composable and makes AI and ML simple, portable, and scalable.
```

Comment:
If we change the first sentence to `Kubeflow is the foundation of tools for AI platforms on Kubernetes.`, we need to adjust the following sentences to make it read well.

Let's avoid repeating the word Kubeflow at the start of each sentence, and simplify the message of the second sentence to `Our tools are modular, portable, and scalable.`

This would give us:

> Kubeflow is the foundation of tools for AI platforms on Kubernetes. Our tools are modular, portable, and scalable. We are an ecosystem of Kubernetes based projects for each stage in the AI/ML Lifecycle with support for best-in-class open source tools and frameworks.

The above text would be acheived by applying this diff:

```suggestion
      Kubeflow is the foundation of tools for AI platforms on Kubernetes. 
      Our tools are modular, portable, and scalable.
```

Reply:
" {{<blocks/cover title="Kubeflow" subtitle="The foundation of tools for AI Platforms on Kubernetes." image_anchor="center" color="dark">}}

Kubeflow is the foundation of tools for AI platforms on Kubernetes. 
 Our platform is composable and our tools are modular, portable, and scalable. We are an ecosystem of Kubernetes based projects for each stage in the AI/ML Lifecycle with support for best-in-class open source tools and frameworks."

sounds good to me.

Reply:
I am not sure if the second sentence should start with "Our Platform".
If we say that "Kubeflow is the foundation of tools for AI Platforms", what do we mean by "Our Platform" ?

Reply:
```suggestion
      Kubeflow is the foundation of tools for AI platforms on Kubernetes. Kubeflow is composable and makes AI and ML simple, portable, and scalable.AI platform teams can build on top of Kubeflow, deploy the entire reference platform, or use each project independently to meet their specific needs. The Kubeflow reference platform is composable, modular, portable, and scalable, backed by an ecosystem of Kubernetes-native projects covering every stage of the AI/ML lifecycle. It integrates best-in-class open source tools and frameworks.
```